Userland/Utilities/unzip.cpp

```diff
@@ -17,6 +18,18 @@
 #include <LibCrypto/Checksum/CRC32.h>
 #include <sys/stat.h>
 
+static ErrorOr<void> adjust_modification_time(Archive::ZipMember const& zip_member)
+{
+    auto time = AK::time_from_packed_dos(zip_member.modification_date, zip_member.modification_time);
+    auto ts = time.to_timespec();
```

Comment:
Did we need to create a timespec here if we're only caring about seconds? we can probably just slice off the seconds with `to_seconds`

```
auto seconds = static_cast<time_t>(time.to_seconds());
```

Perhaps a future refactor could name `static_cast<time_t>(to_seconds())` as `static time_t Time::to_posix_time()`